src/core/Slider/Slider.tsx

```diff
@@ -163,8 +163,8 @@ export type SliderProps = {
    */
   thumbProps?: (index: number) => React.HTMLAttributes<HTMLDivElement>;
   /**
-   * Callback fired when the value(s) of the slider has changed. This will receive
-   * changes at the end of a slide as well as changes from clicks on rails and tracks.
+   * Callback fired at the end of a thumb move and when there user clicks on rail
```

Comment:
maybe enough `and when user clicks on rail`? this 'there' does not give anything..does it?

Reply:
I think that's a typo 😆

Reply:
I was also gonna say, maybe we should explicitly mention `pointerUp` in this comment

Reply:
I wish I could use the excuse that English is my second language.